apps/i18n/common/en_us.json

```diff
@@ -155,6 +156,9 @@
   "changeUserTypeModal_currentEmail_isRequired": "An email address is required.",
   "changeUserTypeModal_currentEmail_label": "Email address",
   "changeUserTypeModal_currentEmail_labelDetails": "(This must match an email address associated with your account.)",
+  "changeUserTypeModal_emailOptIn_description": "Can we email you about updates to our courses, local opportunities, or other computer science news?",
```

Comment:
Worth de-duping with same strings appearing above?

Reply:
I've intentionally kept them separate in case we need to change one without the other at a moment's notice.  For instance, in the original spec the opt-in description in the "Change email" dialog included this text:

> This will not impact your subscription preferences for your old email address.

Latest instructions from Poorva has identical text in these two places, but it's not obvious to me that they'd change together in the future.